src/contextify.cc

```diff
@@ -180,170 +182,201 @@ class ContextWrap : public ObjectWrap {
         return true;
     }
 
-    static NAN_PROPERTY_GETTER(GlobalPropertyGetter) {
-        NanScope();
-        Local<Object> data = args.Data()->ToObject();
+    static void GlobalPropertyGetter(Local<String> property, const PropertyCallbackInfo<Value>& info) {
+        Local<Object> data = info.Data()->ToObject();
         ContextifyContext* ctx = ObjectWrap::Unwrap<ContextWrap>(data)->ctx;
-        if (!ctx)
-            NanReturnUndefined();
-        Local<Value> rv = NanNew(ctx->sandbox)->GetRealNamedProperty(property);
-        if (rv.IsEmpty()) {
-            rv = NanNew(ctx->proxyGlobal)->GetRealNamedProperty(property);
+
+        if (!ctx) {
+            info.GetReturnValue().SetUndefined();
+            return;
         }
-        NanReturnValue(rv);
+
+        Local<Value> rv = Nan::New(ctx->sandbox)->GetRealNamedProperty(property);
+
+//        if (rv.IsEmpty()) {
+//            rv = Nan::New(ctx->proxyGlobal)->GetRealNamedProperty(property);
+//        }
+
+        info.GetReturnValue().Set(rv);
     }
 
-    static NAN_PROPERTY_SETTER(GlobalPropertySetter) {
-        NanScope();
-        Local<Object> data = args.Data()->ToObject();
+    static void GlobalPropertySetter(Local<String> property, Local<Value> value, const PropertyCallbackInfo<Value>& info) {
```

Comment:
Only that it didn't seem to work and I didn't want to spend too much time trying to figure out why, I needed to move fast. But I can try again now that I got Contextify to work with node 5.0.0.

I'd prefer to add the new functionality to NAN and avoid having ifdefs in a client project like Contextify, if you're willing to elaborate.